### PR TITLE
usage with sway

### PR DESCRIPTION
**Pull request by mathroc**
see #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +38,17 @@ To use gpu encoding, use a VAAPI codec (for ex. `h264_vaapi`) and specify a GPU
...
 wf-recorder -f test-vaapi.mkv -c h264_vaapi -d /dev/dri/renderD128
 ```
+
+## with [Sway](https://swaywm.org/)
+
+In your Sway config file, add those key bindings to start and stop wf-recorder :
+
+```
+bindsym Ctrl+Print exec wf-recorder -f ~/recording_$(date +"%Y-%m-%d_%H:%M:%S.mp4")
+bindsym Ctrl+Shift+Print exec wf-recorder -g "$$(slurp)" -f ~/recording_$(date +"%Y-%m-%d_%H:%M:%S.mp4")
```
Comment (soreau) [inline README.md]: Is this double $$ a typo in `-g "$$(slurp)"`?

**Comment by soreau** [comment]
This information might be a better fit for the wf-recorder wiki, just a thought.

File: README.md (hunk 1 above)
Comment (mathroc) [reply README.md]: no it's for escaping the `$`, otherwise it's considered a configuration variable

**Comment by mathroc** [comment]
I don't mind having this in the wiki but I don't think I can edit it. I'll let anyone interested either merging this PR or copying its content in the wiki

**Comment by ammen99** [comment]
I agree we'd better add this to the Wiki page, since in the future we'll want to show integrations with more environments. I'm not sure if people can send PRs to the wiki.

**Comment by norcalli** [comment]
I have a different version which allows you to use a single `bindsym` and a flock to be able to start and stop the video which I keep in a script. Would sharing such a script be a better solution? That way people can integrate with it however they want.

**Comment by soreau** [comment]
@norcalli Can you upload it as a [gist](https://gist.github.com/) and post the link here?

**Comment by ammen99** [comment]
I think we should really have a wiki page where this all belongs to (what this PR has, also what @norcalli has).

Just paste it to gist, and I'll upload it to the wiki (since AFAIK there are no pull requests for the wiki)

**Comment by norcalli** [comment]
@soreau https://gist.github.com/norcalli/81fa49055767d84986637151237f7dbd the recording path is hardcoded to my preferences and uses slurp for region recording. Canceling slurp records the whole screen. Running the script again stops screen recording.

**Comment by soreau** [comment]
> I think we should really have a wiki page where this all belongs to (what this PR has, also what @norcalli has).
> 
> Just paste it to gist, and I'll upload it to the wiki (since AFAIK there are no pull requests for the wiki)

I created a [wiki page](https://github.com/ammen99/wf-recorder/wiki/Usage-with-Sway) with this information.

**Comment by ammen99** [comment]
@soreau Thank you. In this case we can close this PR.
Also thanks @mathroc for giving those tips, I think users will find them useful :)

**Pull request closed without merging**